### DISCUSSION TITLE
/webservice/WebserviceStandard/{moduleName}/RecordsList x-condition JSON format SYNTAX

### DISCUSSION THREAD

**Discussion by root2k6** [YetiForceCompany/YetiForceCRM#17028] [How to] [answered]
I have one question like in subject. 

An example:
{
  "fieldName": "lastname",
  "value": "Kowalski",
  "operator": "e",
  "group": true
}

but, when I want add second condition with AND or OR operator between

{
  "fieldName": "lastname",
  "value": "Kowalski",
  "operator": "e",
  "group": true
}
[ AND / OR ] <- ?????
{
  "fieldName": "email",
  "value": "@domain.com",
  "operator": "c",
  "group": true
}

**Comment by mariuszkrzaczkowski** [comment]
![image](https://user-images.githubusercontent.com/10198654/232419779-bdb82527-db89-417b-b17d-27f2bcc72fe1.png)

https://gitdeveloper.yetiforce.com/index.php?module=Contacts&view=List&viewname=7&page=1&orderby=[]&entityState=Active&mid=49&parent=47&advancedConditions=null&search_params=[[["lastname","a","Kowalski"],["email","a","@domain.com"]]]&totalCount=0

[[["lastname","a","Kowalski"],["email","a","@domain.com"]]]

**Comment by root2k6** [comment]
I understand you, this is search_params in form on page, but how do it in json from x-confition, when I send api request?

**Comment by rskrzypczak** [comment]
Simple use json: https://doc.yetiforce.com/api/#/BaseModule/4c33930681be7788fce3bf019c3a6336
example: json_encode([["lastname","a","Kowalski"],["email","a","@domain.com"]])

**Comment by root2k6** [comment, marked as answer]
A have solution, for everyone, who write API integration in Delphi for example.
query: SELECT * FROM table WHERE name like '%some%' **and** city='city_name'
x-condition JSON:
[{ "fieldName": "name", "value": "some", "operator": "c", "group": **true** },
{ "fieldName": "city", "value": "city_name", "operator": "e", "group": **true** }]

query: SELECT * FROM table WHERE name like '%some%' **or** city='city_name'
x-condition JSON:
[{ "fieldName": "name", "value": "some", "operator": "c", "group": **false** },
{ "fieldName": "city", "value": "city_name", "operator": "e", "group": **false** }]

It works in my application perfect.
